src/PostgREST/MediaType.hs

```diff
@@ -65,6 +66,7 @@ toContentType ct = (hContentType, toMime ct <> charset)
 -- | Convert from MediaType to a ByteString representing the mime type
 toMime :: MediaType -> ByteString
 toMime MTApplicationJSON      = "application/json"
+toMime MTApplicationJSONSQL   = "application/json+sql"
```

Comment:
`application/json` suffixes have to be registered on IANA to be valid ([ref](https://en.wikipedia.org/wiki/Media_type#Suffix)).

> Unregistered suffixes should not be used

So we would need a vendored media type. I suggest `application/vnd.pgrst.sql+json`.

Reply:
> application/sql+json

IIUC, we would have to register the `+json` to the SQL media type if we want to use it like that.

That being said, I like that media type too and it doesn't seem "potentially damaging", unlike adding a suffix to `application/json`.

Reply:
I do like the idea of vendoring the media type, though prefixing `sql` makes it more clear though IMO. Just to be safe, I'll probably go with the vendored media type for now